keps/sig-apps/3998-job-success-completion-policy/README.md

```diff
@@ -650,21 +442,10 @@ to implement this enhancement.
 - Feature implemented behind the `JobSuccessPolicy` feature gate.
 - Unit and integration tests passed as designed in [TestPlan](#test-plan).
 
-#### Optional Second Alpha
-
-- Decided whether we introduce a `whenCriteriaAchived=continue` and `whenCriteriaAchived=continueWithRecreations`.
-Please see the [Possibility for the lingering pods to continue running after the job meets the successPolicy](#possibility-for-the-lingering-pods-to-continue-running-after-the-job-meets-the-successpolicy)
-section for discussion points.
-- Decided whether we introduce a new CronJob concurrentPolicy, `ForbidUntilJobSuccessful`.
-Please see the [Possibility for introducing a new CronJob concurrentPolicy, "ForbidUntilJobSuccessful"](#possibility-for-introducing-a-new-cronjob-concurrentpolicy-forbiduntiljobsuccessful).
-section for more details.
-- Decided whether we introduce a configurable reason for the `SuccessCriteriaMet` condition.
-Please see the [Possibility for the configurable reason for the "SuccessCriteriaMet" condition](#possibility-for-the-configurable-reason-for-the-successcriteriamet-condition). 
-
 #### Beta
 
 - E2E tests passed as designed in [TestPlan](#test-plan).
-- Added a new reasons to the existing `job_finished_total` metric in [Monitoring Requirements](#monitoring-requirements).
+- Introduced a new `job_succeeded_total` metric in [Monitoring Requirements](#monitoring-requirements).
```

Comment:
> Are there any tests for feature enablement/disablement?

nit: You'r mentioning you did an integration test - can you please link it?
[I see it linked below so it's not a blocking issue, but would help for the future]

Reply:
Sure! Thank you for the suggestion!